op-validator/pkg/validations/addresses.go

```diff
@@ -0,0 +1,32 @@
+package validations
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	VersionV180 = "v1.8.0"
+	VersionV200 = "v2.0.0"
+)
+
+var addresses = map[uint64]map[string]common.Address{
+	11155111: {
```

Comment:
At the first glance, this only seems to be for Sepolia. Any reason to have it for mainnet (`1`)?

Reply:
I haven't deployed the contracts to mainnet yet. The contracts are in a state of flux and I didn't want to spend real money deploying them until they stabilized.